src/pages/index.js

```diff
@@ -1,73 +1,57 @@
 import Head from 'next/head';
-import {useEffect} from 'react';
+import styled from 'styled-components';
 
 import Layout from '../components/Layout';
-import MovieList from '../components/MovieList';
-import useStore from '../hooks/useStore';
-
-export default function HomePage() {
+import MovieListSmall from '../components/MovieListSmall';
+import SingleItem from '../components/SingleItem';
+import useFetch from '../hooks/useFetch';
+
+const Ho2 = styled.h2`
+	margin: 32px 16px 0 8px;
+	padding: 12px 8px 16px 12px;
+	color: black;
+	overflow-wrap: break-word;
+	font-size: 24px;
+`;
+
+export default function Movies() {
+	//
 	const API_KEY = process.env.API_KEY;
 
-	const GET_CONFIGURATION = `https://api.themoviedb.org/3/configuration?api_key=${API_KEY}`;
-
-	//const GET_MOVIES = `https://api.themoviedb.org/3/discover/movie?api_key=${process.env.API_KEY}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1`;
+	const GET_MOVIES = `https://api.themoviedb.org/3/discover/movie?api_key=${API_KEY}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1`;
 
 	const GET_TV = `https://api.themoviedb.org/3/discover/tv?api_key=${API_KEY}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1`;
```

Comment:
I would probably rename these to `MOVIE_URL` and `TV_URL` as it is not a `get` function but just the URLs.

Reply:
These are the API calls for get movies and get tv shows. So this is consistent with keeping track of what i'm doing api wise. But yeah, not perfect. 😄

Reply:
But what you are saving in those variables are _not_ the API calls, but only the URLs.